src/screens/one_player_screen/one_player_screen.cpp

```diff
@@ -403,24 +422,18 @@ void OnePlayerScreen::scrollNoteGround(
  *
  */
 void OnePlayerScreen::initSpeedLabel(
-    const sf::RenderWindow &app
+    const sf::RenderWindow& window,
```

Comment:
why renaming it window though it's named `app` in all the other places 

same for the place of the `&`  ;-)